music_assistant/common/models/enums.py

```diff
@@ -56,7 +57,7 @@ class ExternalID(StrEnum):
     UNKNOWN = "unknown"
 
     @classmethod
-    def _missing_(cls: Self, value: object) -> Self:  # noqa: ARG003
+    def _missing_(cls, value: object) -> ExternalID:  # noqa: ARG003
```

Comment:
```suggestion
    def _missing_(cls: type, value: object) -> Self:  # noqa: ARG003
```

maybe this works ?

Reply:
https://github.com/python/mypy/issues/15223 maybe?

Reply:
argh wth is this so convoluted haha. Well, lets just pick what works now and not worry too much about it